substrate/client/consensus/beefy/src/lib.rs

```diff
@@ -388,6 +405,53 @@ where
 		})
 }
 
+async fn wait_for_parent_header<B, BE>(
+	backend: &BE,
+	current: <B as Block>::Header,
+	delay: Duration,
+) -> ClientResult<<B as Block>::Header>
+where
+	B: Block,
+	BE: Backend<B>,
+{
+	let blockchain = backend.blockchain();
```

Comment:
nit: I think you could directly pass `blockchain` reference
```suggestion
async fn wait_for_parent_header<B, BE>(
	blockchain: &BE::Blockchain,
	current: <B as Block>::Header,
	delay: Duration,
) -> ClientResult<<B as Block>::Header>
where
	B: Block,
	BE: Backend<B>,
{
```

Reply:
Done